pkgs/by-name/no/nomnatong/package.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchurl
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "nomnatong";
+  version = "5.07";
+
+  srcs = [
+    (fetchurl {
+      name = "nomnatong-regular.otf";
+      url = "https://github.com/nomfoundation/font/releases/download/v${finalAttrs.version}/nomnatong-regular.otf";
+      hash = "sha256-ojOh6sewZpcGqc8Kg+zeiu0g42MAnLYq1F1BWaFwc2s=";
+    })
+    (fetchurl {
+      url = "https://github.com/nomfoundation/font/releases/download/v${finalAttrs.version}/nomnatong-regular.ttf";
+      hash = "sha256-SCBYZ/A+NllhD+uMtr6WmM/vRbGkejiDxMCHhoaFhp8=";
+    })
+    (fetchurl {
+      url = "https://github.com/nomfoundation/font/releases/download/v${finalAttrs.version}/nomnatonglight.ttf";
```

Maybe?
```suggestion
      name = "nomnatong-light.ttf"
      url = "https://github.com/nomfoundatio
      n/font/releases/download/v${finalAttrs.version}/nomnatonglight.ttf";
```

I'm not sure if this font file _should_ have a `-` in the name or not...